motivaa-backend/motivaa-app/src/test/java/backendtesting/TestUtils.java

```diff
@@ -11,6 +11,57 @@
 
 public class TestUtils {
 
+    public static final String LOCKVERSIONS = "lockVersions";
+    public static final String UPDATEMAP = "updateMap";
+
+
+    public static void updateProcessObjectAndVerifyOutputValues(String environmentHost,
+                                                                String processUpdateEndpoint,
+                                                                String jSessionId,
+                                                                ProcessObject processObject,
+                                                                String inputAttribute,
+                                                                String inputValue,
+                                                                List<OutputVerification> inputOutputExpectedValues) {
+
+            List<Map<String, String>> lockVersions = processObject.getLockVersions();
+            String processUuid = processObject.getProcessUuid();
+
+            Map<String, List<String>> updateMap = TestUtils.createUpdateMapFromAttributeIdAndValue(
+                    inputAttribute,
+                    inputValue);
+
+            Map<String, Object> requestBody = TestUtils.createRequestBodyFromUpdateMapAndLockVersions(
+                    updateMap,
+                    lockVersions);
+
+            String actualUpdateResponse = RequestMaker.updateExistingProcessObject(
```

Comment:
this is a rather long method with multiple responsibilities. I think you could separate them by having one shich does the actual request and an other one which asserts the values in the response. Those can be private methods, which then called by this method here.